src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyAmendmentServicePermissionTests.cs

```diff
@@ -41,7 +41,8 @@ await PolicyCheckBuilder<SecurityPolicies>()
             return new MethodologyAmendmentService(
                 contentPersistenceHelper ?? DefaultPersistenceHelperMock().Object,
                 userService ?? new Mock<IUserService>().Object,
-                MapperUtils.AdminMapper()
+                MapperUtils.AdminMapper(),
+                null
```

Comment:
Could this be a `new Mock<ContentDbContext>(MockBehavior.Strict).Object` rather than null?

Reply:
I don't think DbContexts play nicely with Moq, so I think you either wire in an InMemory one (which I'm not using in these tests) or a null.

Reply:
I've supplied an InMemoryContentDbContext (actually named InMemoryApplicationDbContext, which we must get round to changing at some point!)